views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+          works may vary from country to country. Terms have a start and end
+          date, and the data therefore describes the membership of the
+          legislation during that period. In most democracies, new terms begin
+          after every national election. Of course, changes still occur within
+          terms -- the point is, that’s how we break the data files down for
+          you.
+        </p>
+        <p>
+          This is pragmatic because we know online services and sites are
+          likely to be concerned with the most recent term -- that is, a
+          country’s <em>current</em> legislature.
+        </p>
+        <h2>
+          Politicians
+        </h2>
+        <p>
+          <strong>The richness of the data we have will depend on the source or
+          sources from which we’re getting it.</strong> We try to use official
+          sources, but this isn’t consistently useful across the world. Indeed,
+          the EveryPolitician project exists partly because in some places it’s
+          still depressingly hard to get even basic information about the
+          current legislature.
+        </p>
+        <p>
+          But the absolute minimum data we need for a politician to be in
+          EveryPolitician is their <strong>name</strong> (by implication, we
+          also have their membership of the legislature and its term). Unless
+          the system they are in does not use such mechanisms, we’ll also have
+          the <strong>area</strong> they represent, and the
+          <strong>group</strong> (perhaps a party or faction) they belong to.
+          There may also be start and end dates if we know they did not serve
+          the full term.
+        </p>
+        <a name="about-names"></a>
+        <h3>
+          About names
+        </h3>
+        <p>
+          Names are a key part of any data set based around people. In the
+          basic CVS, the <code>name</code> will be the common name that we’ve
+          got from the source, but we’re aware that names are not quite that
+          simple. We often have this name data:
+        </p>
+        <p>
+          <code>name</code>
+          <code>given_name</code>
+          <code>family_name</code>
+          <code>honorific_prefix</code>
+          <code>honorific_suffix</code>
+          <code>patronymic_name</code>
+          <code>sort_name</code>
+        </p>
+        <p>
+          However, in the CSV you’ll be getting these in whichever language was
+          available from the source of the data. Often this is the official
+          language of the country concerned, but that’s not always
+          straightforward because many countries have multiple official
+          languages. Furthermore, you can’t determine <em>which</em> language
+          it is from within the CSV — remember that CSV really is a coarse
+          version of the data — so instead you should use the JSON. In there
```

Comment:
There's a recurring undercurrent through of "I'm telling you what's in the CSV, but you really shouldn't be using it". We should either go the other way around, describing the JSON, but noting that a simplified version is available as a CSV, or soften these to a much more explicit "if you need X, then the JSON is probably better".